packages/sparql-proxy/lib/fetchServiceDescription.js

```diff
@@ -0,0 +1,12 @@
+import rdf from '@zazuko/env-node'
+
+export default {
+  fetchServiceDescription: async function fetchServiceDescription(endpointUrl, format) {
+    const response = await rdf.fetch(endpointUrl, {
```

Comment:
I would expect to have a way to provide the credentials, in case the endpoint requires auth.

Trifid, with its sparql-proxy plugin is also often used to provide a public endpoint from an authenticated one ; so in this case there would be an issue I think.

Reply:
of course!